<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonSpeakerOpen">
    <property name="geometry">
     <rect>
      <x>216</x>
      <y>128</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonScreenCalibration">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>342</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>屏幕校准</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonScreenTest">
    <property name="geometry">
     <rect>
      <x>286</x>
      <y>340</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>屏幕测试</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSpeakerClose">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>128</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBeepOpen">
    <property name="geometry">
     <rect>
      <x>216</x>
      <y>178</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBeepClose">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>178</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCardClose">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>228</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCardOpen">
    <property name="geometry">
     <rect>
      <x>216</x>
      <y>228</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderSoundSize">
    <property name="geometry">
     <rect>
      <x>436</x>
      <y>138</y>
      <width>160</width>
      <height>19</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>118</y>
      <width>96</width>
      <height>141</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>喇叭测试：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>蜂鸣器测试：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>刷卡测试：</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>HelloZhytek</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>69</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>HelloZhytek</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonEthTest">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ETH测试</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonWifiTest">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Wifi测试</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>wifi名称：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>wifi密码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SD卡：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSDCardStatus">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>挂载成功</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButtonSpeakerOpen</tabstop>
  <tabstop>pushButtonScreenCalibration</tabstop>
  <tabstop>pushButtonBeepClose</tabstop>
  <tabstop>pushButtonBeepOpen</tabstop>
  <tabstop>pushButtonSpeakerClose</tabstop>
  <tabstop>pushButtonScreenTest</tabstop>
  <tabstop>pushButtonCardClose</tabstop>
  <tabstop>pushButtonCardOpen</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
